pontoon/base/static/js/translate.js

```diff
@@ -600,10 +600,11 @@ var Pontoon = (function (my) {
      * callback Callback function
      */
     checkUnsavedChanges: function (callback) {
-      var entity = this.getEditorEntity();
+      var entity = this.getEditorEntity(),
+	      self = this;
 
       // Ignore for anonymous users, for which we don't save translations
-      if (!this.user.email || !entity) {
+      if (!self.user.uid || !entity) {
         this.restoreInPlaceTranslation();
         return callback();
       }
```

Comment:
If we delete this entire block we'll see:
![2016-09-01_02-34-44 2](https://cloud.githubusercontent.com/assets/3993729/18149300/74bbcb48-6ff0-11e6-9b1e-e3f2cdcdd787.png)